### PR TITLE
IODA-60: Generate OpenAPI spec

### PR DESCRIPTION
**Pull request by mdbudd**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,6 +20,7 @@
         "symfony/property-info": "6.2.*",
         "symfony/runtime": "6.2.*",
         "symfony/serializer": "6.2.*",
+        "symfony/twig-bundle": "6.2.*",
```
Comment (bartfeenstra) [inline composer.json:23]: What is Twig being used for here?

Comment (mdbudd) [reply composer.json:23]: A dependency of the openapi spec package

Comment (bartfeenstra) [reply composer.json:23]: We must only declare those packages we use directly as dependencies. If something is a dependency of one of our dependencies, we must not add it ourselves.

File: tests/src/Integration/Controller/ContentNegotiationControllerTest.php [hunk 2]
```diff
@@ -93,4 +93,19 @@ public function testFiveHundred(): void
         $this->assertSame(500, $response->getStatusCode());
         $this->assertJson((string)$response->getContent());
     }
+
+    public function testOpenApi(): void
```
Comment (bartfeenstra) [inline tests/src/Integration/Controller/ContentNegotiationControllerTest.php:97]: This is not a content negotiation test.

**Review by bartfeenstra — CHANGES_REQUESTED** [review]

**Review by bartfeenstra — APPROVED** [review]

**Pull request merged**